### ISSUE TITLE
Docker: Function not implemented: AH00141: Could not initialize random number generator

### ISSUE THREAD

**Issue by dasgarner** [xibosignage/xibo#3099]
OP: https://community.xibo.org.uk/t/function-not-implemented-ah00141-could-not-initialize-random-number-generator/29950

> According to various sources, this is due to an lower kernel-version. Unfortunately, we use Docker on a Synology NAS, wherefore updating the kernel is not possible.
>
> Could there be a image/version which is suitable for kernel-versions of 3.10?

**Comment by alexharrington** [comment]
Kernel 3.10 went out of security support in 2017, so really Synology need to provide a firmware update if the device is still something they support.

The bullseye image comes with an older version of Apache, which is why it then works. I think we could provide a Dockerfile for people on Synology to use perhaps, but I don't think it's sensible to roll back the main release images to an earlier version of Apache.

**Comment by dasgarner** [comment]
Building a separate image for Synology with each release isn't viable, as it takes an age to release already. The instructions for Synology would have to be a set of commands to download the code, build an image, run the image?

**Comment by dasgarner** [comment]
The proposed fix from the OPs thread was to change everything that isn't `buster` to `buster, which as you say is far too old for our main release.

**Comment by alexharrington** [comment]
Agreed, but in the issue they link it says that bullseye also works - just not bookworm.

**Comment by dasgarner** [comment]
As far as I am able to tell the issue with NextCloud was solved by reverting to `buster`.

Regardless, the solution seems to be the same - provide a Dockerfile and instructions for building it.

**Comment by vinnco** [comment]
Any more information about this?

**Comment by dasgarner** [comment]
We don't have a way forward here - the project does not have the resources to maintain a Dockerfile specific for Synology, and we must keep up with security fixes for our main Dockerfile.

The community could come forward with a Dockerfile.
